Packs/Hackuity/Integrations/Hackuity/Hackuity.py

```diff
@@ -0,0 +1,694 @@
+            },
+        )
+        output["VulnType"]["Name"] = client.get_vulnerability_name(finding["vuln_type"])
+        outputs.append(output)
+    return CommandResults(
+        outputs_prefix="Hackuity.Findings",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            "Findings",
+            outputs,
+            headers=["Asset", "VulnType", "Attributes", "Score", "Status"],
+        ),
+    )
+
+
+def hackuity_search_vulnerabilities_command(
+    client: Client, args: Dict[str, Any], hy_global_only: bool
+) -> CommandResults:
+    asset_name = args.get("asset_name")
+    asset_type = args.get("asset_type")
+    attribute = args.get("attribute")
+    limit = int(args["limit"])
+    cvss_min = none_or_apply(args.get("cvss_min"), float)
+    cvss_max = none_or_apply(args.get("cvss_max"), float)
+    vuln_type = args.get("vuln_type")
+    trs_min = none_or_apply(args.get("trs_min"), int)
+    trs_max = none_or_apply(args.get("trs_max"), int)
+    aggfindings = client.fetch_aggfindings(
+        asset_name=asset_name,
+        asset_type=asset_type,
+        attribute=attribute,
+        cvss_min=cvss_min,
+        cvss_max=cvss_max,
+        hy_global_only=hy_global_only,
+        limit=limit,
+        trs_min=trs_min,
+        trs_max=trs_max,
+        vuln_type=vuln_type,
+    )
+    outputs = []
+    for aggfinding in aggfindings:
+        output = json_format(
+            aggfinding,
+            {
+                "attributes": ["Attributes"],
+                "cvss": ["Score", "CVSS"],
+                "description": ["Description"],
+                "id": ["ID"],
+                "seen_first": ["Seen", "First"],
+                "trs": ["Score", "TRS"],
+            },
+            {
+                "seen_first": format_date,
+            },
+        )
+        output["Findings"] = json_format(
+            aggfinding["by_state"],
+            {
+                "total": ["Total"],
+                "open": ["Open"],
+                "closed": ["Closed"],
+                "ignored": ["Ignored"],
+            },
+        )
+        output["VulnTypes"] = [
+            {
+                "ID": vuln_type,
+                "Name": client.get_vulnerability_name(vuln_type),
+            }
+            for vuln_type in aggfinding["vuln_types"]
+        ]
+        outputs.append(output)
+    return CommandResults(
+        outputs_prefix="Hackuity.Vulnerabilities",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            f"{'VulnDB' if hy_global_only else 'Provider'} vulnerabilities",
+            outputs,
+            headers=[
+                "VulnTypes",
+                "Description",
+                "Attributes",
+                "Score",
+                "Findings",
+                "Seen",
+            ],
+        ),
+    )
+
+
+def hackuity_dashboard_widgets_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    widgets = client.fetch_dashboard_widgets()
```

Comment:
in case non is returned (no dashboards) return a readable output stating that, do not fail the command

Reply:
By design the dashboard with the user id (the one we are requesting in `fetch_dashboard_widgets`) will always exist; in case there are no widgets `fetch_dashboard_widgets` will return an empty list, I think it is ok

Reply:
okay, sounds good